.github/workflows/build-linux.yml

```diff
@@ -60,6 +55,18 @@ jobs:
           # error_reporting: https://github.com/zendframework/zend-code/issues/160
           ini-values: variables_order=EGPCS, enable_dl=On, allow_url_fopen=On, error_reporting=-1, memory_limit=1G, date.timezone=UTC, xdebug.max_nesting_level=256
 
+      - name: Set environment variables
+        run: |
+          echo "::set-env name=SYMFONY_PHPUNIT_VERSION::${{ matrix.symfony_phpunit }}"
+          echo "::set-env name=ZEPHIR_PARSER_VERSION::v1.3.3"
+          echo "::set-env name=PHP_INI_DIR::$(php-config --configure-options | tr -s " " "\012" | grep with-config-file-scan-dir | cut -d= -f2)"
+
+      - name: Enable PSR extension
+        run: |
+          if ! php --ri psr 1>/dev/null; then
+            echo 'extension="psr.so"' | sudo tee "$PHP_INI_DIR/psr.ini"
+          fi
```

Comment:
@shivammathur Is this expected?

Reply:
No, this step is not required. `ext-psr` is installed using the package from [`ppa:ondrej/php`](https://launchpad.net/~ondrej/+archive/ubuntu/php) which enables the extension itself upon installation.
Here is a test workflow - https://github.com/shivammathur/test-setup-php/actions/runs/31796827/workflow

Reply:
As you can see in the previous commit of this PR it is not enabled by default, only installed.

Reply:
> Warning: Cannot locate class "Psr\Http\Message\MessageInterface" when extending interface "Test\Integration\Psr\Http\Message

`Psr\Http\Message\MessageInterface` is provided by PSR extension

Reply:
https://github.com/phalcon/zephir/runs/410729035#step:16:217